readme.md

```diff
@@ -1,7 +1,8 @@
 # NixOS COSMIC
 
-Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs.
+Nix package set and NixOS module for using COSMIC from NixOS. This is a temporary repository for testing COSMIC on NixOS as it is developed. When COSMIC gets more stable and it is fully working on NixOS, these packages and module are intended to be merged upstream into nixpkgs(track progress [here](https://github.com/NixOS/nixpkgs/issues/259641).
```

Comment:
This issue doesn't really track progress of this repo. So it should probably have different wording (e.g. "issue to be updated when upstreaming")